src/pages/AllocationPage/AllocationEpoch.tsx

```diff
@@ -133,7 +133,6 @@ const AllocationEpoch = ({
   useEffect(() => {
     if (myUser) {
       setEpochBio(myUser?.bio ?? '');
-      setNonReceiver(myUser.non_receiver);
```

Comment:
Wait. Does this mean if I've opted out and I return to this page, it will show me as opted in?